workflow-templates/sync-git-to-svn.yml

```diff
@@ -0,0 +1,27 @@
+name: Sync-BRANCH-to-svn
+on:
+  push: YOUR_SPECIFIC_BRANCH #NOTE: Works for ONE specific branch
+
+jobs:
+  sync-git-to-svn: # make sure the action works on a clean machine without building
+    runs-on: [self-hosted, linux]
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Use Node.js (.nvmrc)
+        uses: actions/setup-node@v2
+        with:
+          node-version: '${{ steps.nvm.outputs.NVMRC }}'
```

Comment:
The target repo probably has no `.nvmrc` file. I guess we can remove these lines.